src/service/storage/storage.service.ts

```diff
@@ -73,27 +73,23 @@ export class StorageService {
     await this.client.send(command);
   }
 
-  public async getSignedGetUrl(key: string, expiresIn: number): Promise<string> {
+  public async getSignedGetUrl(key: string): Promise<string> {
     const input: GetObjectCommandInput = {
       Bucket: this.s3Config.bucket,
       Key: key,
     };
 
     const command = new GetObjectCommand(input);
 
-    return await getSignedUrl(this.client, command, { expiresIn });
+    return await getSignedUrl(this.client, command, { expiresIn: this.s3Config.linkTtl });
```

Comment:
Can it be useful to separate linkTtl for upload and download? What do you think?

Reply:
I agree, it is better to make different variables for this, as it is likely better to give more time to upload